app/packages/state/src/utils.ts

```diff
@@ -106,30 +106,30 @@ export const transformDataset = (dataset: any): Readonly<State.Dataset> => {
   };
 };
 
-export const getDatasetName = (context: RoutingContext<any>): string => {
+export const getDatasetSlug = (context: RoutingContext<any>): string => {
   const result = matchPath(
     context.pathname,
     {
-      path: "/datasets/:name",
+      path: "/datasets/:slug",
       exact: true,
     },
     {},
     ""
   );
-
-  if (result) {
-    return decodeURIComponent(result.variables.name);
+  const slug = result?.variables?.slug;
```

Comment:
This is guaranteed to be defined, no?

Reply:
yeah just took a look at `matchPath`

I'll just do the `if(result)` like before